src/ngraph/runtime/cpu/cpu_external_function.hpp

```diff
@@ -62,6 +62,11 @@ namespace ngraph
             using OpMap = std::unordered_map<std::type_index, OpFunction>;
 #endif
 
+#define EMIT_DEBUG_MANIFEST(writer, tensor_name)                                                   \
+    {                                                                                              \
```

Comment:
this would be very expensive. Maybe we could consider adding a new member variable `bool print_debug_manifest` to `runtime::cpu::CPU_ExternalFunction`